core/src/main/java/org/eclipse/hono/util/RequestResponseApiConstants.java

```diff
@@ -43,6 +43,37 @@
     public static final String FIELD_PAYLOAD   = "payload";
     public static final String FIELD_TENANT_ID = "tenant-id";
 
+    /* request actions */
+    public enum Action {
+        ACTION_GET, ACTION_ADD, ACTION_UPDATE, ACTION_REMOVE, ACTION_UNKNOWN;
```

Comment:
I do not believe that these are the same for all request/response APIs, are they?

Reply:
Unfortunately: you are right. I haven't looked into the registration API, which defines more actions than this. With the Credentials API it would have been fitting (but only for now).

I will come up with a better solution, this has been too much generalized obviously...